docs/advanceDocumentation.md

```diff
@@ -0,0 +1,24 @@
+# Advance Settings
+
+**Table of Contents**
+
+- [Role based Access Control](#-rbac-settings)
+- [Allow and Deny lists](#-allow--and-deny-list-settings)
+- [Free Pricing](#-free-pricing-settings)
+
+## RBAC settings
+
+- Setup and host the Ocean role based access control (RBAC) server. Follow the instructions in the RBAC repository: https://github.com/oceanprotocol/RBAC-Server
+- The RBAC server can store roles in [Keycloak](https://www.keycloak.org/) or a json file.
+- Create an environmental variable in the .env file called `GATSBY_RBAC_URL`. The value should be the URL of the Ocean RBAC server that you have hosted, e.g. `GATSBY_RBAC_URL= "http://localhost:3000"`
```

Comment:
wee can assume people have already copied `.env.example` to `.env` as thee main readme outlines. So people have that `GATSBY_RBAC_URL` already there so we should modify these instructions here a bit